sdks/java/core/src/test/java/org/apache/beam/sdk/PipelineTest.java

```diff
@@ -407,13 +406,9 @@ public void testReplaceAll() {
         new PipelineVisitor.Defaults() {
           @Override
           public CompositeBehavior enterCompositeTransform(Node node) {
-            if (!node.isRootNode()) {
-              assertThat(
-                  node.getTransform().getClass(),
-                  not(
-                      anyOf(
-                          Matchers.equalTo(GenerateSequence.class),
-                          Matchers.equalTo(Create.Values.class))));
+            String fullName = node.getFullName();
+            if (fullName.equals("unbounded") || fullName.equals("bounded")) {
+              assertThat(node.getTransform(), Matchers.instanceOf(EmptyFlatten.class));
```

Comment:
Could you explain this fix a bit?

Reply:
Rather than checking what the transform class after applying the replacement should not be (which was never the case anyways), this asserts that the resulting transform is an instance of the expected class